package.json

```diff
@@ -5,15 +5,20 @@
     "email": "roger@bufferoverflow.ch"
   },
   "scripts": {
+    "build": "babel src/ --out-dir lib/ --copy-files --ignore ___tests___ --source-maps",
```

Comment:
`--source-maps` I'd remove that, otherwise you will inflate your package.

Reply:
the map files are not packaged with the npm, but they are very useful to debug, I keep it.